### PR TITLE
Axis label

### PR DESCRIPTION
**Pull request by Alanocallaghan**
This is a minor edit; it just corrects the y axis label for `ExIPlot`s

### CODE REVIEW

File: R/visualization.R [hunk 1]
```diff
@@ -3604,7 +3604,7 @@ ExIPlot <- function(
     FUN = function(x) {
       return(SingleExIPlot(
         type = type,
-        data = data[, x, drop = FALSE],
+        data = data[, features, drop = FALSE],
```
Comment (andrewwbutler) [inline R/visualization.R]: Why change this to `features`? I believe `SingleExIPlot` expects a single column data.frame.

Comment (alanocallaghan) [reply R/visualization.R]: That's unintended, I meant to stash and discard that. I'll remove

Comment (alanocallaghan) [reply R/visualization.R]: Done, should be up to date with devel also

**Comment by andrewwbutler** [comment]
Thanks for the fix!

**Comment by Alanocallaghan** [comment]
Thanks for the quick response!

**Pull request merged**